server/src/lfortran-accessors.ts

```diff
@@ -356,16 +379,11 @@ export class LFortranCLIAccessor implements LFortranAccessor {
         const range: Range = location.range;
 
         const start: Position = range.start;
-        // if uri contains .ts file, then we need not to decrement the line and character by 1
-        if (!uri.includes(".ts")) {
-          start.line--;
-        }
+        start.line--;
```

Comment:
@Pranavchiku is the `uri.includes(".ts")` line needed?

Reply:
`npm test` uses that, but if the CI is passing I think it shall be fine.

Reply:
The CI failed after I merged it with the above stuff removed. Should it be added in?

Reply:
I didn't realize a commit was merged into main, I've fixed it in the latest commit. The "*.ts" checks should not be needed since the extension should only be run against Fortran files. I've removed the condition in the latest commit but let me know if they need to be added back.

Reply:
>  The "*.ts" checks should not be needed since the extension should only be run against Fortran files.

`npm test` uses `uri` as `*.ts` file.

Reply:
Also the Fortran files that we use in `npm test` have `0` based indexing unlike VSCode so we cannot do `start.line--` for those files.

Reply:
Just put @Pranavchiku's quick fix of "uri.includes(".ts")" back in, get CI working, merge.

Then you can investigate a better solution, that will be a minor PR. We need the fixes in this PR in.

Reply:
To get things moving, I put it back in in baf628afefbc75a1a2698920601ae78654cea9fc. Let's see if CI passes now.